app.js

```diff
@@ -18,19 +18,28 @@ let mySQLSessionConfig = {
   port: config.port,
   user: config.user,
   password: config.password,
-  database: config.database
+  database: config.database,
+  nodeEnv: config.nodeEnv
  }
  
- var sessionStore = new expressMySQLSession(mySQLSessionConfig);
+  var sessionStore = new expressMySQLSession(mySQLSessionConfig);  
```

Comment:
Thinking about it, this line, and the definition of mySQLSessionConfig on line 16 could be moved into the if statement on line 36, so even the initialisation of the MySQLStore is only done if running in a production environment.